cmd/boot-script-service/main.go

```diff
@@ -443,6 +445,27 @@ func main() {
 		}
 	}
 
+	// register oauth client and receive
+	var client OAuthClient
+	_, err = client.RegisterOAuthClient("http://127.0.0.1:4444/oauth2/register", []string{})
+	if err != nil {
+		log.Fatalf("failed to register OAuth client: %v", err)
+	}
+	_, err = client.AuthorizeClient("http://127.0.0.1:4444/oauth2/auth")
+	if err != nil {
+		log.Fatalf("failed to authorize OAuth client: %v", err)
+	}
+	res, err := client.FetchTokenFromAuthorizationServer("http://127.0.0.1:4444/oauth2/token", []string{})
+	if err != nil {
+		log.Fatalf("failed to fetch token from authorization server: %v", err)
+	}
```

Comment:
Looks like localhost is hardcoded. Can we set this with flag/environment variable?

Reply:
Fixed with [c1ec85f](https://github.com/OpenCHAMI/bss/pull/23/commits/c1ec85fa7e3f946168e9dc716205befe0574e120).